### PR TITLE
[Docs](README) Change of Contribution Guidelines

### PR DESCRIPTION
**Pull request by jonathino2590**
# Tasks / Issue :construction:
```
It was found that the link to the contribution guide is badly referenced
```

# Solution :heavy_check_mark:
```
[ksctl](https://github.com/kubesimplify/ksctl)/CONTRIBUTING.md

```

# Screenshots :framed_picture:

image

# Note to reviewers :notebook:

### CODE REVIEW

File: build/Dockerfile [hunk 1]
```diff
@@ -32,6 +32,5 @@ RUN mkdir -p ${HOME}/.ksctl/cred && \
     mkdir ${HOME}/.ksctl/config/aws && \
     mkdir ${HOME}/.ksctl/config/local
 
-RUN wget -O kubectl https://dl.k8s.io/release/v1.25.0/bin/linux/amd64/kubectl
-
-RUN chmod +x kubectl && mv ./kubectl /usr/local/bin/kubectl
+RUN wget -O kubectl https://dl.k8s.io/release/v1.25.0/bin/linux/amd64/kubectl && \
+    chmod +x kubectl && mv ./kubectl /usr/local/bin/kubectl
```
Comment (dipankardas011) [inline build/Dockerfile:36]: @jonathino2590 we should not make 2 RUN's as it increases the number of Layers in Docker build thus increasing the image size

Comment (jonathino2590) [reply build/Dockerfile:36]: Correct that is the change that is proposed

**Comment by AvineshTripathi** [comment]
Just a suggestion can we keep only guidelines changes over here and create dockerfile changes to another PR

File: build/Dockerfile (hunk 1 above)
Comment (dipankardas011) [reply build/Dockerfile:36]: But it will degrade the Docker build time and space @kranurag7

**Comment by jonathino2590** [comment]
Ok 👍

**Comment by dipankardas011** [comment]
> Just a suggestion can we keep only guidelines changes over here and create dockerfile changes to another PR 

# Recommended 
Do changes as docs in docs pr
@jonathino2590

**Comment by jonathino2590** [comment]
I close this PR and open the following https://github.com/kubesimplify/ksctl/pull/41

**Pull request closed without merging**